### PR TITLE
Fix filter drawer, eslint and removed prettier

### PR DESCRIPTION
**Pull request by Blemming**
Didnt seperate commits very well, my bad.
Fixed up the filter drawer so it uses the expansion panels a bit better
Setup antfus eslint
removed prettier

I have not linted the whole project yet as this would probably make for a huge PR to look through, but I'll fix up the formatting in files as I get to them.

### CODE REVIEW

**Comment by FloKaemmerer** [comment]
I will have a close look on the PR tomorrow (likely before the exilecon stream)

One thing I noticed wile scrolling through on mobile:
The linter removes the {} on single line if.
Maybe it's a java thing or the bit of clean code that stuck with me, I'd like to always use braces for if's. It's cleaner, more consistent and less error prone.

**Comment by Blemming** [comment]
Here you go, I overrided the rule and reapplied it to affected code :)

File: src/components/filter_drawer/FilterDrawer.vue [hunk 1]
```diff
@@ -55,79 +31,117 @@ onMounted(() => {
...
+
   if (state.minEffectiveDivinationCardValue > 0) {
     queryFilters.minEffectiveDivinationCardValue = state.minEffectiveDivinationCardValue
   }
-  //---------------------------
 
-  filterQueryStore.SET_FILTER_QUERY(queryFilters)
+  // ---------------------------
 
-  pushToRouter(queryFilters).then(() => {
```
Comment (FloKaemmerer) [inline src/components/filter_drawer/FilterDrawer.vue:124]: not calling this breaks the url being updated with the active Filters.

File: src/components/filter_drawer/FilterDrawer.vue [hunk 2]
```diff
@@ -1,133 +1,100 @@
...
+const filterQueryStore = useFilterQueryStore()
+const filterDrawerStore = useFilterDrawerStore()
+const route: RouteLocationNormalizedLoaded = useRoute()
+const router: Router = useRouter()
 
 const drawer = computed<boolean>(() => filterDrawerStore.drawer)
 
-onMounted(() => {
-  // We need to import the AtlasFilter composable, otherwise it won't trigger, even though it is subscribed to the FilterStore
+function queryFilters() {
```
Comment (FloKaemmerer) [inline src/components/filter_drawer/FilterDrawer.vue:27]: That looks way better! Thank you!

(Firsttime really working with JS/TS let alone Vue, and it shows ;-))

**Review by FloKaemmerer — APPROVED** [review]

**Pull request merged**